substrate/frame/election-provider-multi-block/src/signed/tests.rs

```diff
@@ -518,11 +525,26 @@ mod e2e {
 
 			roll_to_signed_validation_open();
 
+			// Clear verifier events to track incremental verification
+			let _ = verifier_events();
```

Comment:
```suggestion
```

is not doing anything. Sometimes it is useful to call

```
/// flush the events so far
let _ = verifier_events_since_last_call()
```

because `verifier_events_since_last_call` sets the counter internally.

The test indeed passes without this.